bundles/org.openhab.binding.sonnen/README.md

```diff
@@ -0,0 +1,58 @@
+# Sonnen Binding
+
+The binding for sonnen communicates with a sonnen battery. More information about the sonnen battery can be found [here](https://sonnen.de/)
+
+## Supported Things
+
+| Things | Description  | Thing Type |
+|--------|--------------|------------|
+| sonnen | Control of a sonnen battery | battery|
+
+
+## Thing Configuration
+
+Only one parameter is required. The IP-Address of the sonnen battery in your local network.
+
+```
+Thing sonnen:battery:myBattery "sonnenBattery"  [ hostIP="192.168.0.10"]
+```
+
+## Channels
+
+The following channels are yet supported:
+
+
+| Channel | Type  | Access| Description|
+|---------|-------|-------|------------|
+|batteryCharging|Switch|read|Indicates if the Battery is charging at that moment|
```

Comment:
In the documentation, it is the channel id that is expected by the user.
In another comment, I recommend to you to use `batteryCharging` as channel id.
in case you will not change the channel id, you have to use `BatteryCharging` in the documentation.

Reply:
changed it already, forgot it there